### ISSUE TITLE
 The reason why depth map should be divided 2**16 ? 

### ISSUE THREAD

**Issue by yingfei1016** [EPFL-VILAB/MultiMAE#11]
Thank you for your great Multi-MAE, We observed that in https://github.com/EPFL-VILAB/MultiMAE/blob/main/utils/datasets.py line 96, you use  img = torch.Tensor(np.array(task_dict[task]) / 2 ** 16). Can you tell me the reason why depth map should be divided 2**16 ? Is there any problems without this operation?

**Comment by alexsax** [comment]
@woctezuma is right, this ensures that the images are in the range [0,1]. The images in the Omnidata/Taskonomy datasets are stored on disk as single-channel 16-bit PNG images, so depth is a uint16 in {0,..2**16-1}. This corresponds to a [min, max] of [0,128] meters

Technically I suppose the normalization factor should be 2**16-1, but this is a difference of 1/512m which is well under the amount of noise in the labels themselves, which is on the order of centimeters.

**Comment by dmizr** [comment]
Hi @yingfei1016, 
This scaling factor is indeed there to ensure that the depth images are in the range [0, 1]. 
Note that this does not matter for pre-training as we standardize the depth afterwards.
https://github.com/EPFL-VILAB/MultiMAE/blob/8ea9be251e8e39d1a68fea9c7a68ef1797e9df62/run_pretraining_multimae.py#L487

**Comment by RunpeiDong** [comment]
Thanks for the great work and the release of the depth pseudo-label. I wonder if the depth pseudo-labels are in centimeters? What are the camera intrinsics for those depth images or RGB images?

**Comment by roman-bachmann** [comment]
Hi @RunpeiDong, 

The depth pseudo-labels are not metric, as the Omnidata model we used to pseudo-label them was trained with a scale- and shift invariant loss (see [Omnidata](https://omnidata.vision/) and [MiDaS](https://arxiv.org/abs/1907.01341)). 
Regarding camera intrinsics: We pseudo-labeled ImageNet-1K, so as far as I am aware, camera parameters are not available.

Best, Roman
